README.md

```diff
@@ -100,8 +92,19 @@ sudo cp /usr/local/devcycle/dataCollection/settings.py.template /usr/local/devcy
 sudo vim /usr/local/devcycle/dataCollection/settings.py
  ```
  * Under DATABASES, modify USER and PASSWORD fields to reflect the database user you created in ["Install The Database"](#install-postgres-with-postgis-install-the-database).
+ * Change "TIME_ZONE" to match the timezone of the server in “tz database” format (e.g. ‘America/New_York’)
+ * Change "DEBUG" to `True` if you are **not** in production
+ * Change "DEFAULT_MAP_LAT" and "DEFAULT_MAP_LON" to match the latitude and longitude, respectively, that maps in the dashboard should default to.
+ * Change "MAP_TILE_SERVER" to match the hostname of the server to retrieve map tiles from (currently configured to use MapQuest's free OSM tile server)
+
+8. Set up logging
+
+ ```
+sudo mkdir /var/log/devcycle-logs
+sudo chown -R www-data:www-data /var/log/devcycle-logs
```

Comment:
alright, for the noting ubuntu: should we comment that in the setup file or is the label at the top of the readme stating that this is for ubuntu 14.04 enough?